### ISSUE TITLE
easter egg problem

### ISSUE THREAD

**Issue by GAMERMEER** [ShadowMario/FNF-PsychEngine#13462]
### Describe your problem here.

i got for some reason the easter egg while i was working on my mod but how can i remove it completly from the game?
<img width="960" alt="image" src="https://github.com/ShadowMario/FNF-PsychEngine/assets/113830775/6843b048-1abc-480c-b0b2-470e42de2d6b">

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

yes i changed some files like the TitleState.hx but it was to add some text in the intro.

**Comment by BernardoGP4504** [comment]
Hey umm, that logo ain't a Easter egg, you just made it visible. Simply remove the code for it

**Comment by GAMERMEER** [comment]
> Hey umm, that logo ain't a Easter egg, you just made it visible. Simply remove the code for it

well it isn't i just compiled the game and that showed up without touching anything other titlestate cooltext code

**Comment by BernardoGP4504** [comment]
> > Hey umm, that logo ain't a Easter egg, you just made it visible. Simply remove the code for it
> 
> well it isn't i just compiled the game and that showed up without touching anything other titlestate cooltext code

you see, in the code there's 2 variables, logo and logoBl. Simply remove logo and keep logoBl

**Comment by GAMERMEER** [comment]
> > > Hey umm, that logo ain't a Easter egg, you just made it visible. Simply remove the code for it
> > 
> > 
> > well it isn't i just compiled the game and that showed up without touching anything other titlestate cooltext code
> 
> you see, in the code there's 2 variables, logo and logoBl. Simply remove logo and keep logoBl
yeah and then hope that the game doesn't crash

**Comment by GAMERMEER** [comment]
AGAIN 
[PsychEngine_2023-10-18_17'51'49.txt](https://github.com/ShadowMario/FNF-PsychEngine/files/13030780/PsychEngine_2023-10-18_17.51.49.txt)

**Comment by BernardoGP4504** [comment]
The game will crash if you remove `logoBl` or if you remove the `logo.png` from the images folder without remove the `logo` sprite first

**Comment by BernardoGP4504** [comment]
Remove lines 300 to 303

**Comment by BernardoGP4504** [comment]
> Remove lines 300 to 303

![Screenshot_20231018-174013-481](https://github.com/ShadowMario/FNF-PsychEngine/assets/101405920/d02ea1f9-51dd-4182-b06f-9718d5348953)

**Comment by GAMERMEER** [comment]
> The game will crash if you remove `logoBl` or if you remove the `logo.png` from the images folder without remove the `logo` sprite first

no it was doing that before

**Comment by GAMERMEER** [comment]
[TitleState is in here.zip](https://github.com/ShadowMario/FNF-PsychEngine/files/13045262/TitleState.is.in.here.zip)

**Comment by BernardoGP4504** [comment]
Hmm that's weird 

> AGAIN [PsychEngine_2023-10-18_17'51'49.txt](https://github.com/ShadowMario/FNF-PsychEngine/files/13030780/PsychEngine_2023-10-18_17.51.49.txt)

It crashed on loading the logoBumpin image

**Comment by BernardoGP4504** [comment]
Hmm that's weird, I don't see any code for that logo

**Comment by GAMERMEER** [comment]
> > AGAIN [PsychEngine_2023-10-18_17'51'49.txt](https://github.com/ShadowMario/FNF-PsychEngine/files/13030780/PsychEngine_2023-10-18_17.51.49.txt)
> 
> It crashed on loading the logoBumpin image

but it's just 20 frames and on the builded psych engine with the mods folder it just loads fine

**Comment by BernardoGP4504** [comment]
Is the file on assets/preload/images ?

**Comment by GAMERMEER** [comment]
> Is the file on assets/preload/images ?

yes it's in the correct location and i tested it before in the mods folder before i added it in in that location

**Comment by GAMERMEER** [comment]
so everything was ok but then after i got that weird logo and i was trying to remove it and it didn't work then i used ctrl + z to undo everything then that happend

**Comment by BernardoGP4504** [comment]
Well maybe the Ctrl+Z made it worse. Can't you just go to an earlier commit of your Title state.hx?
